content/post/career-ladders-part-2.md

```diff
@@ -0,0 +1,56 @@
+---
+author: Caitlin Moorman
+title: "How to Create a Career Ladder"
```

Comment:
I'm not sure where it goes in this post, but this was a lesson I learned the hard way: more junior team members tend to want to see more formal progression early in their careers. So if you are hiring people straight out of undergrad, you will want to make sure that your career ladder (and compensation plan) allows for the ability to give them one or two promotions within their first few years. After that, expect the timelines to stretch...